api/v1/flow/flow.proto

```diff
@@ -387,6 +387,9 @@ message FlowFilter {
     // node_name is a list of patterns to filter on the node name, e.g. (e.g.
     // "k8s*", "test-cluster/*.domain.com", "cluster-name/").
     repeated string node_name = 24;
+
+    // filter based on IP version (ipv4 or ipv6)
+    repeated IPVersion ip_version = 25;
```

Comment:
I don't think this particular filter needs to be repeated since it's effectively a binary choice

Reply:
Is it possible that someone specifies --ipv4 and --ipv6 both? ... just because their Hubble API calls are designed that way? (I understand that the default is both)

Reply:
> [...] since it's effectively a binary choice

Only until the next IP version. More seriously, I'm neutral about this.

Reply:
It's already not a binary choice. The field already has three values: No IP Version, IPv4, IPv6. I think it can be very legitimate to filter by flows which have `IPVersion_IP_NOT_USED` (e.g. ARP packets).

Reply:
ok, then I believe keeping repeated here makes sense? Can I mark this resolved?